### PR TITLE
Adding Drupal and devops

### PR DESCRIPTION
**Pull request by iAhmadZain**
Issue: #3 

### Slack channel name
- Drupal
- Amazeeio
- Wodby 
- Traefik 

### Category
- Dev
- DevOps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - [Kotlin lang](http://slack.kotlinlang.org/)
 - [ASP.NET Core](https://aspnetcore.slack.com/). Sign up [here](http://tattoocoder.com/aspnet-slack-sign-up/)
 - [Frontend Developers](http://frontenddevelopers.org/)
+- [Drupal](https://drupal.slack.com)
```
Comment (AlaaAttya) [inline README.md]: @iAhmadZain plz add the link where the users can request getting an invitation for this community

**Review by AlaaAttya — CHANGES_REQUESTED** [review]

**Comment by iAhmadZain** [comment]
@AlaaAttya  done could you check please ?

**Comment by AlaaAttyaMohamed** [comment]
@iAhmadZain good add the invitation links for these communities too

[amazeeio.io](https://amazeeio.slack.com)
[Wodby](https://wodby-community.slack.com)
[Traefik](https://traefik.slack.com)

**Comment by AlaaAttya** [comment]
@iAhmadZain good add the invitation links for these communities too

amazeeio.io
Wodby
Traefik

**Comment by iAhmadZain** [comment]
@AlaaAttya  Done ;)

**Comment by AlaaAttya** [comment]
done, thanks @iAhmadZain 💯 👍

**Pull request merged**